src/natcap/invest/urban_nature_access.py

```diff
@@ -273,7 +273,7 @@
                     'type': 'number',
                     'units': u.meter,
                     'required':
-                        f'search_radius_mode == "{RADIUS_OPT_POP_GROUP}"',
+                        f'search_radius_mode.value == "{RADIUS_OPT_POP_GROUP}"',
```

Comment:
I think it's not necessary to include the conditional requirement on the column here, since the table as a whole has the same requirement. And it might not have any effect because we don't yet evaluate conditional requirement on columns within a table ( that would be great to add, though!)

Reply:
Good point about not needing this conditional requirement in this parameter because the whole table is required!  I have removed it, and I also implemented conditional requirement for vector fields and CSV rows/columns.